manifests/reposync-crd.yaml

```diff
@@ -1187,30 +1187,17 @@ spec:
                   values:
                     description: values to use instead of default values that accompany
                       the chart. Format values the same as default values.yaml. These
-                      values will take precedence if valuesFileRefs is also specified.
-                      How to handle multiple valuesFiles is determined by `valuesFileApplyStrategy`.
+                      values will take precedence if `valuesFileRefs` is also specified.
                     x-kubernetes-preserve-unknown-fields: true
-                  valuesFileApplyStrategy:
-                    default: override
-                    description: valuesFileApplyStrategy specifies the strategy for
-                      handling multiple valueFiles. It refers to how valuesFiles are
-                      applied if multiple valuesFile define the same key. Can be 'override'
-                      or 'listConcatenate'. 'override' (default) results in the duplicated
-                      keys in later files to override the value from earlier files.
-                      This is equivalent to passing in multiple valuesFiles to Helm
-                      CLI. 'listConcatenate' results in the duplicated keys that are
-                      list elements to have the lists concatenated together before
-                      being used to render the chart.
-                    enum:
-                    - override
-                    - listConcatenate
-                    type: string
                   valuesFileRefs:
                     description: valuesFileRefs holds references to objects in the
                       cluster that represent values to use instead of default values
-                      that accompany the chart. Currently, only ConfigMaps are supported.
-                      Objects listed later will take precedence. How to handle multiple
-                      valuesFiles is determined by `valuesFileApplyStrategy`.
+                      that accompany the chart. Currently, only ConfigMaps are supported,
+                      and the ConfigMap must be in the same namespace as the RootSync
+                      or RepoSync. When multiple valuesFiles are used, duplicated
+                      keys in later files will override the value from earlier files.
+                      This is equivalent to passing in multiple valuesFiles to Helm
+                      CLI. If `values` is also specified, `values` will take precedence.
```

Comment:
Suggest:
If `values` is also specified, fields from `values` will override fields from `valuesFileRefs`.

"precedence" is mildly confusing. This should make it clear that it uses the same override mechanism, not a full replacement.

Reply:
done